keps/sig-network/1860-kube-proxy-IP-node-binding/README.md

```diff
@@ -189,138 +189,71 @@ Yes. It is tested by `TestUpdateServiceLoadBalancerStatus` in pkg/registry/core/
 
 ### Rollout, Upgrade and Rollback Planning
 
-<!--
-This section must be completed when targeting beta to a release.
--->
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-<!--
-Try to be as paranoid as possible - e.g., what if some components will restart
-mid-rollout?
+As the rollout will enable a feature not being used yet, there is no possible failure
+scenario as this feature will then need to be also enabled by the cloud provider on
+the services resources.
 
-Be sure to consider highly-available clusters, where, for example,
-feature flags will be enabled on some API servers and not others during the
-rollout. Similarly, consider large clusters and how enablement/disablement
-will rollout across nodes.
--->
+In case of a rollback, kube-proxy will also rollback to the default behavior, switching 
+back to VIP mode. This can fail for workloads that may be already relying on the 
+new behavior (eg. sending traffic to the LoadBalancer expecting some additional
+features, like PROXY and TLS Termination as per the Motivations section).
 
 ###### What specific metrics should inform a rollback?
 
-<!--
-What signals should users be paying attention to when the feature is young
-that might indicate a serious problem?
--->
+N/A
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-<!--
-Describe manual testing that was done and the outcomes.
-Longer term, we may want to require automated upgrade/rollback tests, but we
-are missing a bunch of machinery and tooling and can't do that now.
--->
+No.
```

Comment:
We should test that before enabling Beta by default.
Can you please describe the scenario here now (and ensure that you actually run it - manual run is fine) before enabling beta?

Reply:
Will do next here, will keep this thread open and close once I have results

Reply:
@wojtek-t did the test here. Just a node (that I have added to the KEP), because this feature is mostly related to cloud providers, the test I did was, on a 3 node KinD cluster, enabled the feature flag, installed MetalLB as the LB, ran some tests, and disabled the Feature flag to check how kube-proxy and apiserver would behave.

It is registered on the steps, let me know if you need something else.